ApeWorX/ape
# feat: test command

### What I did

`ape test` command (synchronous)

### How I did it

Copied stuff from fubu's branch.
Copied stuff from brownie.

### How to verify it

Checkout https://github.com/unparalleled-js/ape-demo-project
Run `ape test`.
The tests should run and pass.

It should compile before testing if needbe.

Note: make sure you have geth installed.

Note: unable to easily add tests using geth as the provider.
Once we are supporting London fork changes, we can add all sorts of tests a lot easier using `EthTester` as the provider, which is great for CI/CD.

Note: xdist is going to be a follow up

### Checklist
